app/user.go

```diff
@@ -839,7 +839,7 @@ func (a *App) SetDefaultProfileImage(user *model.User) *model.AppError {
 		return err
 	}
 
-	<-a.Srv.Store.User().ResetLastPictureUpdate(user.Id)
+	a.Srv.Store.User().ResetLastPictureUpdate(user.Id)
```

Comment:
I think here we can just capture the error and log it if is not nil with `mlog.Error` .

Reply:
Thanks! Makes sense to log the error.